app-drawer/app-drawer.html

```diff
@@ -137,7 +139,8 @@
       transition: opacity 0.2s ease;
 
       -webkit-transform: translateZ(0);
-      transform:  translateZ(0);
+      transform: translateZ(0);
+      will-change: opacity;
```

Comment:
Discussed offline with @frankiefu - we want to avoid using both will-change and the translate3d hack. See https://github.com/PolymerLabs/app-layout/issues/127#issuecomment-200095747